mempool/src/shared_mempool/tasks.rs

```diff
@@ -69,8 +69,17 @@ pub(crate) async fn execute_broadcast<NetworkClient, TransactionValidator>(
                 )
                 .peer(&peer)
                 .error(&error)),
+                BroadcastError::NoTransactions(_) | BroadcastError::PeerNotPrioritized(_, _) => {
+                    sample!(
+                        SampleRate::Duration(Duration::from_secs(60)),
```

Comment:
nit: you could consider making this a const, e.g.:

```
const BROADCAST_LOG_FREQ_SECS: u64 = 60;
```

Reply:
Do we need to sample trace? Just sampling overhead for something that we don't use quite often.